### DISCUSSION TITLE
Preserve the size of "Open Database Object" window

### DISCUSSION THREAD

**Discussion by informatik01** [dbeaver/dbeaver#23213] [Ideas]
Hi.

It would be really convenient if the developers would implement functionality for preserving the size, especially the width, of the "Open Database Objects..." window (top menu "Navigate > Open Database Objects..."). When I open it, it is always has the same size even if I previously changed it (made wider):
<img width="380" alt="image" src="https://github.com/dbeaver/dbeaver/assets/11339880/14b75049-d1c0-4b74-a588-3286ce25c478">

I was expecting that if I change the size / width of this window, next time when I open it - it will be of the same width. Most of the time search results do not fully fit the window width (because tables are shown with the related fully qualified names / schemas), that is why I almost always have to increase the width of this window to have an _aesthetically pleasant_ view of the results.

Please consider this post as a suggestion for the future improvements of the already great product.

---
### P.S.
Today I have discovered the there is a dedicated context menu option (right click on any column) to make ALL (!) the columns fit their values. So I don't have to click each and every column that don't fit its value, but I can do it for all the columns in one go - just wow, what a nice feature!
<img width="229" alt="image" src="https://github.com/dbeaver/dbeaver/assets/11339880/2f979fe9-86b1-4d89-9194-3d0233d33c3d">

Thank you guys for that!
Such little, sometimes subtle nuances make the product really shine.

**Comment by LonwoLonwo** [comment]
Hello @informatik01 

It is better to create a feature request ticket for that idea here:  https://github.com/dbeaver/dbeaver/issues

**Reply by informatik01** [reply]
Hi, @LonwoLonwo.

Created the feature request:
https://github.com/dbeaver/dbeaver/issues/23244#issue-2208994404

**Comment by informatik01** [comment]
Closing this discussion, because I've created a [feature request](https://github.com/dbeaver/dbeaver/issues/23244#issue-2208994404) for that.
